File: src/main/kotlin/com/sordle/WatpadsCardDeck/Player.java
package Demo.Models;

import Demo.RockPaperScissors;
import Demo.RockPaperScissors.RESULT;

import java.util.ArrayList;


public class Player {
  public ArrayList<RockPaperScissors.CARDS> hand;
  public int score = 0;
  public RockPaperScissors.CARDS move;
  public RockPaperScissors.CARDS winningCard;
  public boolean played;

  private int id; // should  match value in the database

  public Player(ArrayList<RockPaperScissors.CARDS> playerDeck){
    this.hand = playerDeck;
    played = false;
  }

  public Player(){
    this.hand = new ArrayList<>();
  }

  public boolean equals(Player other) {
    return this.id == other.id;
  }

  
}

File: src/main/kotlin/com/sordle/WatpadsCardDeck/Player.kt
package com.WatpadsCardDeck
import States.Cards

class Player(
  val: id: Integer,
  val: playerDeck: MutableList<Cards>
  ) {
   val score: Integer = 0;
   val move: Cards;
   val winCondition: Cards;
   val hasPlayed: boolean = false;


  @Synchronized
  fun incrimentScore() {
    score++
  }

  fun equals(other: Player) {
    return other.getID() == getID()
  }

}
